<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="MSI" failures="10" tests="40" name="Test" time="49.667" errors="0" timestamp="2024-08-17T17:09:12 IST">
  <testcase classname="apiTestCases.GetUserInfoTest" name="validateUserInfoTest" time="2.198">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.GetUserInfoTest" name="validateUserInfoTest" time="1.175">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.GetUserInfoTest" name="validateUserInfoTest" time="1.164">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.GetUserInfoTest" name="validateUserInfoTest" time="1.159">
    <failure type="java.lang.RuntimeException" message="com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field &amp;quot;error&amp;quot; (class pojo.UserDetails), not marked as ignorable (3 known properties: &amp;quot;name&amp;quot;, &amp;quot;count&amp;quot;, &amp;quot;age&amp;quot;])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails[&amp;quot;error&amp;quot;])">
      <![CDATA[java.lang.RuntimeException: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "error" (class pojo.UserDetails), not marked as ignorable (3 known properties: "name", "count", "age"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails["error"])
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:30)
at io.restassured.path.json.mapping.JsonPathObjectDeserializer$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.Jackson2Mapper.deserialize(Jackson2Mapper.groovy:58)
at io.restassured.mapper.ObjectMapper$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.parseWithJackson2(ObjectMapping.groovy:254)
at io.restassured.internal.mapping.ObjectMapping$parseWithJackson2$0.callStatic(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.deserialize(ObjectMapping.groovy:56)
at io.restassured.internal.mapping.ObjectMapping$deserialize.call(Unknown Source)
at io.restassured.internal.RestAssuredResponseOptionsGroovyImpl.as(RestAssuredResponseOptionsGroovyImpl.groovy:229)
at io.restassured.internal.RestAssuredResponseOptionsImpl.as(RestAssuredResponseOptionsImpl.java:169)
at helpers.GetUserInfoAPI.ValidateUserInfo(GetUserInfoAPI.java:34)
at apiTestCases.GetUserInfoTest.validateUserInfoTest(GetUserInfoTest.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "error" (class pojo.UserDetails), not marked as ignorable (3 known properties: "name", "count", "age"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails["error"])
at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61)
at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:1153)
at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:2241)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1793)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1771)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:316)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177)
at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:342)
at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4905)
at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3848)
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:28)
... 39 more
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.GetUserInfoTest" name="validateUserInfoTest" time="1.134">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.GetUserInfoTest" name="validateUserInfoTest" time="1.179">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.GetUserInfoTest" name="validateUserInfoTest" time="1.104">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.GetUserInfoTest" name="validateUserInfoTest" time="1.142">
    <failure type="java.lang.RuntimeException" message="com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field &amp;quot;error&amp;quot; (class pojo.UserDetails), not marked as ignorable (3 known properties: &amp;quot;name&amp;quot;, &amp;quot;count&amp;quot;, &amp;quot;age&amp;quot;])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails[&amp;quot;error&amp;quot;])">
      <![CDATA[java.lang.RuntimeException: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "error" (class pojo.UserDetails), not marked as ignorable (3 known properties: "name", "count", "age"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails["error"])
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:30)
at io.restassured.path.json.mapping.JsonPathObjectDeserializer$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.Jackson2Mapper.deserialize(Jackson2Mapper.groovy:58)
at io.restassured.mapper.ObjectMapper$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.parseWithJackson2(ObjectMapping.groovy:254)
at io.restassured.internal.mapping.ObjectMapping$parseWithJackson2$0.callStatic(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.deserialize(ObjectMapping.groovy:56)
at io.restassured.internal.mapping.ObjectMapping$deserialize.call(Unknown Source)
at io.restassured.internal.RestAssuredResponseOptionsGroovyImpl.as(RestAssuredResponseOptionsGroovyImpl.groovy:229)
at io.restassured.internal.RestAssuredResponseOptionsImpl.as(RestAssuredResponseOptionsImpl.java:169)
at helpers.GetUserInfoAPI.ValidateUserInfo(GetUserInfoAPI.java:34)
at apiTestCases.GetUserInfoTest.validateUserInfoTest(GetUserInfoTest.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "error" (class pojo.UserDetails), not marked as ignorable (3 known properties: "name", "count", "age"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails["error"])
at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61)
at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:1153)
at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:2241)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1793)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1771)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:316)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177)
at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:342)
at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4905)
at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3848)
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:28)
... 39 more
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.GetUserInfoTest" name="validateUserInfoTest" time="1.081">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.GetUserInfoTest" name="validateUserInfoTest" time="1.196">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.GetUserInfoTest" name="validateUserInfoTest" time="1.145">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.GetUserInfoTest" name="validateUserInfoTest" time="1.176">
    <failure type="java.lang.RuntimeException" message="com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field &amp;quot;error&amp;quot; (class pojo.UserDetails), not marked as ignorable (3 known properties: &amp;quot;name&amp;quot;, &amp;quot;count&amp;quot;, &amp;quot;age&amp;quot;])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails[&amp;quot;error&amp;quot;])">
      <![CDATA[java.lang.RuntimeException: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "error" (class pojo.UserDetails), not marked as ignorable (3 known properties: "name", "count", "age"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails["error"])
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:30)
at io.restassured.path.json.mapping.JsonPathObjectDeserializer$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.Jackson2Mapper.deserialize(Jackson2Mapper.groovy:58)
at io.restassured.mapper.ObjectMapper$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.parseWithJackson2(ObjectMapping.groovy:254)
at io.restassured.internal.mapping.ObjectMapping$parseWithJackson2$0.callStatic(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.deserialize(ObjectMapping.groovy:56)
at io.restassured.internal.mapping.ObjectMapping$deserialize.call(Unknown Source)
at io.restassured.internal.RestAssuredResponseOptionsGroovyImpl.as(RestAssuredResponseOptionsGroovyImpl.groovy:229)
at io.restassured.internal.RestAssuredResponseOptionsImpl.as(RestAssuredResponseOptionsImpl.java:169)
at helpers.GetUserInfoAPI.ValidateUserInfo(GetUserInfoAPI.java:34)
at apiTestCases.GetUserInfoTest.validateUserInfoTest(GetUserInfoTest.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "error" (class pojo.UserDetails), not marked as ignorable (3 known properties: "name", "count", "age"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails["error"])
at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61)
at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:1153)
at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:2241)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1793)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1771)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:316)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177)
at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:342)
at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4905)
at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3848)
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:28)
... 39 more
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.GetUserInfoTest" name="validateUserInfoTest" time="1.124">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.GetUserInfoTest" name="validateUserInfoTest" time="1.119">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.GetUserInfoTest" name="validateUserInfoTest" time="1.182">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.GetUserInfoTest" name="validateUserInfoTest" time="1.156">
    <failure type="java.lang.RuntimeException" message="com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field &amp;quot;error&amp;quot; (class pojo.UserDetails), not marked as ignorable (3 known properties: &amp;quot;name&amp;quot;, &amp;quot;count&amp;quot;, &amp;quot;age&amp;quot;])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails[&amp;quot;error&amp;quot;])">
      <![CDATA[java.lang.RuntimeException: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "error" (class pojo.UserDetails), not marked as ignorable (3 known properties: "name", "count", "age"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails["error"])
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:30)
at io.restassured.path.json.mapping.JsonPathObjectDeserializer$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.Jackson2Mapper.deserialize(Jackson2Mapper.groovy:58)
at io.restassured.mapper.ObjectMapper$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.parseWithJackson2(ObjectMapping.groovy:254)
at io.restassured.internal.mapping.ObjectMapping$parseWithJackson2$0.callStatic(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.deserialize(ObjectMapping.groovy:56)
at io.restassured.internal.mapping.ObjectMapping$deserialize.call(Unknown Source)
at io.restassured.internal.RestAssuredResponseOptionsGroovyImpl.as(RestAssuredResponseOptionsGroovyImpl.groovy:229)
at io.restassured.internal.RestAssuredResponseOptionsImpl.as(RestAssuredResponseOptionsImpl.java:169)
at helpers.GetUserInfoAPI.ValidateUserInfo(GetUserInfoAPI.java:34)
at apiTestCases.GetUserInfoTest.validateUserInfoTest(GetUserInfoTest.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "error" (class pojo.UserDetails), not marked as ignorable (3 known properties: "name", "count", "age"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails["error"])
at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61)
at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:1153)
at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:2241)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1793)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1771)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:316)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177)
at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:342)
at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4905)
at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3848)
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:28)
... 39 more
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.GetUserInfoTest" name="validateUserInfoTest" time="1.106">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.GetUserInfoTest" name="validateUserInfoTest" time="1.158">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.GetUserInfoTest" name="validateUserInfoTest" time="1.212">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.GetUserInfoTest" name="validateUserInfoTest" time="1.155">
    <failure type="java.lang.RuntimeException" message="com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field &amp;quot;error&amp;quot; (class pojo.UserDetails), not marked as ignorable (3 known properties: &amp;quot;name&amp;quot;, &amp;quot;count&amp;quot;, &amp;quot;age&amp;quot;])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails[&amp;quot;error&amp;quot;])">
      <![CDATA[java.lang.RuntimeException: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "error" (class pojo.UserDetails), not marked as ignorable (3 known properties: "name", "count", "age"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails["error"])
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:30)
at io.restassured.path.json.mapping.JsonPathObjectDeserializer$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.Jackson2Mapper.deserialize(Jackson2Mapper.groovy:58)
at io.restassured.mapper.ObjectMapper$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.parseWithJackson2(ObjectMapping.groovy:254)
at io.restassured.internal.mapping.ObjectMapping$parseWithJackson2$0.callStatic(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.deserialize(ObjectMapping.groovy:56)
at io.restassured.internal.mapping.ObjectMapping$deserialize.call(Unknown Source)
at io.restassured.internal.RestAssuredResponseOptionsGroovyImpl.as(RestAssuredResponseOptionsGroovyImpl.groovy:229)
at io.restassured.internal.RestAssuredResponseOptionsImpl.as(RestAssuredResponseOptionsImpl.java:169)
at helpers.GetUserInfoAPI.ValidateUserInfo(GetUserInfoAPI.java:34)
at apiTestCases.GetUserInfoTest.validateUserInfoTest(GetUserInfoTest.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "error" (class pojo.UserDetails), not marked as ignorable (3 known properties: "name", "count", "age"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails["error"])
at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61)
at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:1153)
at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:2241)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1793)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1771)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:316)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177)
at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:342)
at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4905)
at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3848)
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:28)
... 38 more
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.ValidateUserInfoTest" name="validateUserInfoTest" time="1.15">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.ValidateUserInfoTest" name="validateUserInfoTest" time="1.094">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.ValidateUserInfoTest" name="validateUserInfoTest" time="1.148">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.ValidateUserInfoTest" name="validateUserInfoTest" time="1.215">
    <failure type="java.lang.RuntimeException" message="com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field &amp;quot;error&amp;quot; (class pojo.UserDetails), not marked as ignorable (3 known properties: &amp;quot;name&amp;quot;, &amp;quot;count&amp;quot;, &amp;quot;age&amp;quot;])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails[&amp;quot;error&amp;quot;])">
      <![CDATA[java.lang.RuntimeException: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "error" (class pojo.UserDetails), not marked as ignorable (3 known properties: "name", "count", "age"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails["error"])
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:30)
at io.restassured.path.json.mapping.JsonPathObjectDeserializer$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.Jackson2Mapper.deserialize(Jackson2Mapper.groovy:58)
at io.restassured.mapper.ObjectMapper$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.parseWithJackson2(ObjectMapping.groovy:254)
at io.restassured.internal.mapping.ObjectMapping$parseWithJackson2$0.callStatic(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.deserialize(ObjectMapping.groovy:56)
at io.restassured.internal.mapping.ObjectMapping$deserialize.call(Unknown Source)
at io.restassured.internal.RestAssuredResponseOptionsGroovyImpl.as(RestAssuredResponseOptionsGroovyImpl.groovy:229)
at io.restassured.internal.RestAssuredResponseOptionsImpl.as(RestAssuredResponseOptionsImpl.java:169)
at helpers.GetUserInfoAPI.ValidateUserInfo(GetUserInfoAPI.java:34)
at apiTestCases.ValidateUserInfoTest.validateUserInfoTest(ValidateUserInfoTest.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "error" (class pojo.UserDetails), not marked as ignorable (3 known properties: "name", "count", "age"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails["error"])
at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61)
at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:1153)
at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:2241)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1793)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1771)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:316)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177)
at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:342)
at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4905)
at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3848)
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:28)
... 39 more
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.ValidateUserInfoTest" name="validateUserInfoTest" time="1.148">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.ValidateUserInfoTest" name="validateUserInfoTest" time="1.145">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.ValidateUserInfoTest" name="validateUserInfoTest" time="1.138">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.ValidateUserInfoTest" name="validateUserInfoTest" time="1.156">
    <failure type="java.lang.RuntimeException" message="com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field &amp;quot;error&amp;quot; (class pojo.UserDetails), not marked as ignorable (3 known properties: &amp;quot;name&amp;quot;, &amp;quot;count&amp;quot;, &amp;quot;age&amp;quot;])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails[&amp;quot;error&amp;quot;])">
      <![CDATA[java.lang.RuntimeException: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "error" (class pojo.UserDetails), not marked as ignorable (3 known properties: "name", "count", "age"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails["error"])
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:30)
at io.restassured.path.json.mapping.JsonPathObjectDeserializer$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.Jackson2Mapper.deserialize(Jackson2Mapper.groovy:58)
at io.restassured.mapper.ObjectMapper$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.parseWithJackson2(ObjectMapping.groovy:254)
at io.restassured.internal.mapping.ObjectMapping$parseWithJackson2$0.callStatic(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.deserialize(ObjectMapping.groovy:56)
at io.restassured.internal.mapping.ObjectMapping$deserialize.call(Unknown Source)
at io.restassured.internal.RestAssuredResponseOptionsGroovyImpl.as(RestAssuredResponseOptionsGroovyImpl.groovy:229)
at io.restassured.internal.RestAssuredResponseOptionsImpl.as(RestAssuredResponseOptionsImpl.java:169)
at helpers.GetUserInfoAPI.ValidateUserInfo(GetUserInfoAPI.java:34)
at apiTestCases.ValidateUserInfoTest.validateUserInfoTest(ValidateUserInfoTest.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "error" (class pojo.UserDetails), not marked as ignorable (3 known properties: "name", "count", "age"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails["error"])
at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61)
at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:1153)
at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:2241)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1793)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1771)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:316)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177)
at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:342)
at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4905)
at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3848)
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:28)
... 39 more
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.ValidateUserInfoTest" name="validateUserInfoTest" time="1.107">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.ValidateUserInfoTest" name="validateUserInfoTest" time="1.196">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.ValidateUserInfoTest" name="validateUserInfoTest" time="1.186">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.ValidateUserInfoTest" name="validateUserInfoTest" time="1.193">
    <failure type="java.lang.RuntimeException" message="com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field &amp;quot;error&amp;quot; (class pojo.UserDetails), not marked as ignorable (3 known properties: &amp;quot;name&amp;quot;, &amp;quot;count&amp;quot;, &amp;quot;age&amp;quot;])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails[&amp;quot;error&amp;quot;])">
      <![CDATA[java.lang.RuntimeException: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "error" (class pojo.UserDetails), not marked as ignorable (3 known properties: "name", "count", "age"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails["error"])
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:30)
at io.restassured.path.json.mapping.JsonPathObjectDeserializer$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.Jackson2Mapper.deserialize(Jackson2Mapper.groovy:58)
at io.restassured.mapper.ObjectMapper$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.parseWithJackson2(ObjectMapping.groovy:254)
at io.restassured.internal.mapping.ObjectMapping$parseWithJackson2$0.callStatic(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.deserialize(ObjectMapping.groovy:56)
at io.restassured.internal.mapping.ObjectMapping$deserialize.call(Unknown Source)
at io.restassured.internal.RestAssuredResponseOptionsGroovyImpl.as(RestAssuredResponseOptionsGroovyImpl.groovy:229)
at io.restassured.internal.RestAssuredResponseOptionsImpl.as(RestAssuredResponseOptionsImpl.java:169)
at helpers.GetUserInfoAPI.ValidateUserInfo(GetUserInfoAPI.java:34)
at apiTestCases.ValidateUserInfoTest.validateUserInfoTest(ValidateUserInfoTest.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "error" (class pojo.UserDetails), not marked as ignorable (3 known properties: "name", "count", "age"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails["error"])
at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61)
at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:1153)
at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:2241)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1793)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1771)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:316)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177)
at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:342)
at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4905)
at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3848)
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:28)
... 39 more
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.ValidateUserInfoTest" name="validateUserInfoTest" time="1.163">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.ValidateUserInfoTest" name="validateUserInfoTest" time="1.092">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.ValidateUserInfoTest" name="validateUserInfoTest" time="1.148">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.ValidateUserInfoTest" name="validateUserInfoTest" time="1.075">
    <failure type="java.lang.RuntimeException" message="com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field &amp;quot;error&amp;quot; (class pojo.UserDetails), not marked as ignorable (3 known properties: &amp;quot;name&amp;quot;, &amp;quot;count&amp;quot;, &amp;quot;age&amp;quot;])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails[&amp;quot;error&amp;quot;])">
      <![CDATA[java.lang.RuntimeException: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "error" (class pojo.UserDetails), not marked as ignorable (3 known properties: "name", "count", "age"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails["error"])
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:30)
at io.restassured.path.json.mapping.JsonPathObjectDeserializer$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.Jackson2Mapper.deserialize(Jackson2Mapper.groovy:58)
at io.restassured.mapper.ObjectMapper$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.parseWithJackson2(ObjectMapping.groovy:254)
at io.restassured.internal.mapping.ObjectMapping$parseWithJackson2$0.callStatic(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.deserialize(ObjectMapping.groovy:56)
at io.restassured.internal.mapping.ObjectMapping$deserialize.call(Unknown Source)
at io.restassured.internal.RestAssuredResponseOptionsGroovyImpl.as(RestAssuredResponseOptionsGroovyImpl.groovy:229)
at io.restassured.internal.RestAssuredResponseOptionsImpl.as(RestAssuredResponseOptionsImpl.java:169)
at helpers.GetUserInfoAPI.ValidateUserInfo(GetUserInfoAPI.java:34)
at apiTestCases.ValidateUserInfoTest.validateUserInfoTest(ValidateUserInfoTest.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "error" (class pojo.UserDetails), not marked as ignorable (3 known properties: "name", "count", "age"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails["error"])
at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61)
at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:1153)
at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:2241)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1793)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1771)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:316)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177)
at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:342)
at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4905)
at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3848)
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:28)
... 39 more
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.ValidateUserInfoTest" name="validateUserInfoTest" time="1.178">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.ValidateUserInfoTest" name="validateUserInfoTest" time="1.205">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.ValidateUserInfoTest" name="validateUserInfoTest" time="1.146">
    <skipped/>
  </testcase> <!-- validateUserInfoTest -->
  <testcase classname="apiTestCases.ValidateUserInfoTest" name="validateUserInfoTest" time="1.121">
    <failure type="java.lang.RuntimeException" message="com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field &amp;quot;error&amp;quot; (class pojo.UserDetails), not marked as ignorable (3 known properties: &amp;quot;name&amp;quot;, &amp;quot;count&amp;quot;, &amp;quot;age&amp;quot;])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails[&amp;quot;error&amp;quot;])">
      <![CDATA[java.lang.RuntimeException: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "error" (class pojo.UserDetails), not marked as ignorable (3 known properties: "name", "count", "age"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails["error"])
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:30)
at io.restassured.path.json.mapping.JsonPathObjectDeserializer$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.Jackson2Mapper.deserialize(Jackson2Mapper.groovy:58)
at io.restassured.mapper.ObjectMapper$deserialize.call(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.parseWithJackson2(ObjectMapping.groovy:254)
at io.restassured.internal.mapping.ObjectMapping$parseWithJackson2$0.callStatic(Unknown Source)
at io.restassured.internal.mapping.ObjectMapping.deserialize(ObjectMapping.groovy:56)
at io.restassured.internal.mapping.ObjectMapping$deserialize.call(Unknown Source)
at io.restassured.internal.RestAssuredResponseOptionsGroovyImpl.as(RestAssuredResponseOptionsGroovyImpl.groovy:229)
at io.restassured.internal.RestAssuredResponseOptionsImpl.as(RestAssuredResponseOptionsImpl.java:169)
at helpers.GetUserInfoAPI.ValidateUserInfo(GetUserInfoAPI.java:34)
at apiTestCases.ValidateUserInfoTest.validateUserInfoTest(ValidateUserInfoTest.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "error" (class pojo.UserDetails), not marked as ignorable (3 known properties: "name", "count", "age"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 11] (through reference chain: pojo.UserDetails["error"])
at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61)
at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:1153)
at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:2241)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1793)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1771)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:316)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177)
at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:342)
at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4905)
at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3848)
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:28)
... 38 more
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- validateUserInfoTest -->
</testsuite> <!-- Test -->
